src/components/block/FooterCTACard/FooterCTACard.module.css

```diff
@@ -0,0 +1,28 @@
+.body {
+  color: var(--color-white);
+  font: var(--font-body-medium);
+  line-height: 1.3;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 15px;
+  max-width: 350px;
+  text-align: center;
+}
+
+.button {
+  margin-top: 30px;
+  text-align: center;
+}
+
+.container {
+  height: 400px;
+  width: 480px;
+}
+
+.subtitle {
+  color: var(--color-white);
+  font: var(--font-subtitle);
+  line-height: 1.3;
+  padding-top: 109.5px;
```

Comment:
This exact and fixed padding of 109.5px was probably not Derek's original design intent, since 1) most browsers/devices can't even draw half pixels and 2) this is just a strange number. I can see that you got this number directly from the Design System Figma for this component, but if you look at [the designs for the whole pages of the site](https://www.figma.com/file/kKaWvJvyzhNS9hD21v7Ei5/ShelterTech.org-2020?node-id=1673%3A1602), you can see that not all the cards have the same amount of text in them, specifically that some have two lines of body text and others have three.

The offset from the top of the component is not the same between the different components, but what is consistent about all three instances of this card is that they are all vertically centered. I think what you probably want to do is make the container a flex parent and vertically center all of these items in between.